docs/tutorials/deploy-vm/vappconfig.md

```diff
@@ -1,3 +1,89 @@
 # Deploy a VM with vAppConfig
 
-// TODO ([github.com/vmware-tanzu/vm-operator#123](https://github.com/vmware-tanzu/vm-operator/issues/123))
+This page reviews the bootstrap method vAppConfig.
+
+## Description
+The vAppConfig bootstrap method is useful for legacy VM images that rely on bespoke, boot-time processes that leverage vAppConfig properties for customizing a guest.
+
+## Purpose
+As a DevOp, you can assign Golang-based template strings to vAppConfig values to send information into the guest that is not known ahead of time, such as the IPAM data settings used to configure the VM's network stack.
+
+## Example
+```
+apiVersion: vmoperator.vmware.com/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: legacy-vm
+  namespace: test-ns
+spec:
+  className: best-effort-small
+  imageName: haproxy-v0.2.0
+  powerState: poweredOn
+  storageClass: wcpglobal-storage-profile
+  vmMetadata:
+    secretName: my-secret
+    transport: vAppConfig
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
+  namespace: test-ns
+stringData:
+  nameservers: "{{ (index .V1alpha1.Net.Nameservers 0) }}"
+  hostname: "{{ .V1alpha1.VM.Name }} "                
+  management_ip: "{{ (index (index .V1alpha1.Net.Devices 0).IPAddresses 0) }}"
+  management_gateway: "{{ (index .V1alpha1.Net.Devices 0).Gateway4 }}"
+```
+1.  :wave: The Golang based template string is a representation of [vm-operator virtualmachinetempl_types v1alpha1 api](https://github.com/vmware-tanzu/vm-operator/blob/25fb865e615d377192a870583ab32973e9fbd32a/api/v1alpha1/virtualmachinetempl_types.go#L4)
```

Comment:
Also for the bullets to appear on separate lines, you will need to add a blank line between the two.